docs/markdown/connecting-to-testnet.md

```diff
@@ -0,0 +1,61 @@
+# Connecting to an ILP Testnet
+
+This tutorial will help you start moneyd locally on your computer and connect it to an ILP router on the testnet.
+
+This tutorial describes how to:
+
+1. Configure and start [`moneyd`](https://github.com/interledgerjs/moneyd) to connect to an 
```

Comment:
Should the first mention of `moneyd` (see above) include the link instead?

Reply:
https://github.com/interledger/interledger.github.io/pull/108/commits/259f9fdeeda358cb53f45bb23c88eab35d44af39#diff-95b449acc2e692458b91372a5b6e33e7R3